DiscordBot/commands/mud/attack.js

```diff
@@ -2,6 +2,9 @@ const { DEBUG } = require('../../globals.js');
 const commando = require('discord.js-commando');
 const db = require('../../../dbhandler');
 
+// experience counter
+var xp = 0;
```

Comment:
this shouldn't be a global

Reply:
also, use let instead of var. It's a scoping thing, might want to read up on it.

Reply:
you can put this in the combat loop tbh